src/datanode/src/sql.rs

```diff
@@ -45,6 +46,7 @@ pub enum SqlRequest {
     ShowTables(ShowTables),
     DescribeTable(DescribeTable),
     Explain(Box<Explain>),
+    Delete(DeleteRequest),
```

Comment:
What about passing the Delete statement? So we don't need to put catalog/schema/table name in the DeleteRequest.